app/assets/javascripts/components/toolbar-dropdown.js

```diff
@@ -13,12 +13,17 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     var $buttons = this.$module.querySelectorAll('.app-c-toolbar-dropdown__button')
     var $links = this.$module.querySelectorAll('.app-c-toolbar-dropdown__link')
 
+    if (!this.hasDetailsSupport()) {
```

Comment:
Just to clarify this against the commit message: do you mean that "since we've dropped govuk-frontend we now need to do the polyfill ourselves"?

Reply:
Yes, now govuk-frontend doesn't enforce the polyfill for all details elements (it's being applied only for the ones with a `data-module` attached to them - which is now generated by the details component in govuk_publishing_components). This helps us to have a custom script for the toolbar-dropdown which doesn't clash with the previous polyfill.